.changeset/loud-signs-see.md

```diff
@@ -0,0 +1,11 @@
+---
+"@lynx-js/runtime-wrapper-webpack-plugin": patch
+---
+
+Add `window` variable to callback argument in `background.js` and the `window` is `undefined` in Lynx. Sometimes it's useful to distinguish between Lynx and the Web.
+
+```js
+define('background.js', (..., window) => {
+  console.log(window); // `undefined` in Lynx
```

Comment:
I'm okay with this change, but could you explain the difference between using `if (typeof window === 'undefined')` and this?

Reply:
Some users may write the code like below previously:
```
if (!window) {
  // do something
}
```
We may need to be compatible with their code.